docs/UserGuide.md

```diff
@@ -0,0 +1,168 @@
+# User Guide
+
+This guide explains how to update the [Find Employer Schemes](https://find-employer-schemes.education.gov.uk/) (FES) website, by editing the content in the Content Management System (CMS). This is primarily a guide for content authors/approvers, but will also be useful for developers and testers.
+
+## Contentful CMS
+
+[Contentful](https://www.contentful.com) is the CMS used to edit the content for the website. Log in to Contentful with your given credentials.
+
+## 2i and roles
+
+The content team are given both the 'author' and 'editor' roles. This means every team member can both edit and publish content. It is up to the team, to ensure that all content edits are reviewed by someone other than the author, before publishing.
+
+Developers and testers are given the administrator role, so that they can edit the schema and make use of the test and development environments (see [Environments](#environments)).
+
+## Editing
+
+For a general guide to Contentful, consult the [Contentful help center](https://www.contentful.com/help/).
+
+Some customisations of the out of the box editing experience have been made and are detailed here:
+
+#### Call to action box
+
+As there is no formatting support in Contentful for a call to action box, we instead mark the content we want to appear in a call to action box in a blockquote, with a given first line of `<cta>`.
+
+#### Links
+
+Links can be added in the normal way for Contentful.
+
+If you need a link to open in a new tab, append `(opens in new tab)` to the end of the link text (as recommended by GDS), and the web site will render the link so that it opens a new tab, if clicked.
+
+#### Embedded YouTube videos
+
+If you want to embedd a YouTube video, copy the embed code for the YouTube video, making sure to select "Enable privacy-enhanced mode", and copy that into an edit box. Here's a [guide to get the embed code](https://axbom.com/embed-youtube-videos-without-cookies/).
+
```

Comment:
Do we need to mention that the renderer will convert youtube embed urls to cookieless ones anyway?

Reply:
Guide updated as suggested.